gpu/gosl/slprint/comment.go

```diff
@@ -0,0 +1,155 @@
+// Copyright 2022 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package slprint
```

Comment:
We need all of this stuff in `slprint`? There are no other packages that provide some of this functionality?